internal/server/mcp.go

```diff
@@ -63,12 +75,34 @@ func mcpHandler(s *Server, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// return empty response
-	res := mcp.JSONRPCResponse{
-		Jsonrpc: mcp.JSONRPC_VERSION,
-		Id:      baseMessage.Id,
-		Result:  mcp.EmptyResult{},
+	// Check if message is a notification
+	if baseMessage.Id == nil {
+		var notification mcp.JSONRPCNotification
+		if err := json.Unmarshal(body, &notification); err != nil {
+			render.JSON(w, r, newJSONRPCError(baseMessage.Id, mcp.PARSE_ERROR, err.Error(), nil))
+		}
+		// Notifications do not expect a response
```

Comment:
nit: add comment that says "toolbox doesn't do anything with notifications yet, so we ignore them"